source/includes/_api.md

```diff
@@ -28,11 +28,12 @@ var additionalIDs = {
 var configParams = {
   disableBeforeUnloadHandler: true, // disables callstats.js's window.onbeforeunload parameter.
   applicationVersion: "app_version", // Application version specified by the developer.
-  disablePrecalltest: true // disables the pre-call test, it is enabled by default.
+  disablePrecalltest: true // disables the pre-call test.
   siteID: "siteID", // The name/ID of the site/campus from where the call/pre-call test is made.
   additionalIDs: additionalIDs, // additionalIDs object, contains application related IDs.
   collectLegacyStats: true //enables the collection of legacy stats in chrome browser
   collectIP: true //enables the collection localIP address
+  cdnAuth: true //Uses the CDN based URL for contacting Authentication Servers. 
```

Comment:
Probably just "Use CDN for authentication"

Reply:
Uses the CDN for authentication instead of the authentication within the region associated with your organisation.